source/simulator/assembly.cc

```diff
@@ -984,6 +1012,14 @@ namespace aspect
 
 
 
+  template <int dim>
+  Simulator<dim>::Assemblers::Properties::Properties ()
+    :
+    need_face_material_model_data (false)
```

Comment:
No, the error happens because an `FEValues` complains that it can't take an `update_normals` flags (only `FE(Sub)?FaceValues` objects can take this flag). I'm dumbfounded why GCC complains but clang doesn't.